src/main/java/org/testng/internal/ClassBasedParallelWorker.java

```diff
@@ -67,6 +67,10 @@
     return result;
   }
 
+  private static boolean isSame(Set<Class<?>> sequentialClasses, Class<?> c) {
```

Comment:
`isChild` ?
or `isSubclass` (with reversed args)?

Reply:
or like an extension `isAssignableFrom`

or `contains` if we consider it like an overriding of the `contains` from Collection

Reply:
I renamed the method to `shouldRunSequentially()` and moved the contains logic into it.